js/widgets/page.js

```diff
@@ -158,15 +140,12 @@ $.widget( "mobile.page", {
 		}
 	},
 
-	_setOptions: function( o ) {
-		if ( o.theme !== undefined ) {
-			this.element.removeClass( "ui-page-theme-" + this.options.theme ).addClass( "ui-page-theme-" + o.theme );
-		}
-
-		if ( o.contentTheme !== undefined ) {
-			this.element.find( "[data-" + $.mobile.ns + "='content']" ).removeClass( "ui-body-" + this.options.contentTheme )
-				.addClass( "ui-body-" + o.contentTheme );
+	_setOptions: function( newOptions ) {
```

Comment:
since we are changing this whole function anyway and it really makes no sense for a single option the way it is, lets switch this to follow the UI convention of `setOption` vs `setOptions`